File: LeetCode/0007. Reverse Integer/C#/01. Reverse Integer.cs
int Reverse(int x)
{
    if (x > 0)
    {
        char[] chars = x.ToString().ToCharArray();
        Array.Reverse(chars);
        if (Int32.MaxValue >= Int64.Parse(new string(chars)))
            return int.Parse(new string(chars));
        else
            return 0;
    }
    else
    {
        char[] chars = x.ToString().Replace("-", string.Empty).ToCharArray();
        Array.Reverse(chars);
        if (Int32.MaxValue >= Int64.Parse(new string(chars)))
            return int.Parse((new string(chars)).Insert(0, "-"));
        else
            return 0;
    }
}

File: LeetCode/0007. Reverse Integer/JavaScript/01. Reverse Integer.js
var reverse = function (x) {
    if (x > 0) {
        let chars = x.toString().split("").reverse().join("");
        let intResult = parseInt(chars);
        if (intResult <= Math.pow(2, 31) - 1) {
            return intResult;
        }
        else {
            return 0;
        }
    }
    else {
        let chars = x.toString().replace("-", "").split("").reverse().join("");
        let intResult = parseInt("-" + chars);
        if (intResult >= -(Math.pow(2, 31) - 1)) {
            return intResult;
        }
        else {
            return 0;
        }
    }
}